docs/application/dotnet/guides/user-interface/nui/ui-components.md

```diff
@@ -11,29 +11,33 @@ The following table lists the available UI components:
 
 **Table: UI components**
 
-| Control                              | Description                              | Related classes                          |
+| Control                              | Description                              | Appearance                          |
```

Comment:
Comment: All XAML page links are not working. The issue has been raised and discussed. However, it is yet to be resolved. 
@mijin85cho we'd appreciate your input on this.